zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreter.java

```diff
@@ -89,6 +91,7 @@ public RemoteInterpreter(Properties property,
     this.interpreterPath = interpreterPath;
     this.env = env;
     this.connectTimeout = connectTimeout;
+    this.maxPoolSize = 10;
```

Comment:
This constructor is used by tests.
Since zeppelin-interpreter is an upstream dependency for zeppelin-zengine, i am not able to use the value of `ZEPPELIN_INTERPRETER_MAX_POOL_SIZE` from `ConfVars`.
Hence hard coding to 10.